### DISCUSSION TITLE
New models that are still not on the epdiy table and what are the risks/ challenges to take

### DISCUSSION THREAD

**Discussion by martinberlin** [vroland/epdiy#298] [General]
I've seen in the latest days many requests to drive new panels that are not officially supported and not listed in the table.
Basically you need two things to drive any panel:

1. The right FPC connector, pinout and voltages. Voltages are mostly working for all panels the same with some exceptions if the FET Gate needs an unusually high voltage (Even like that some still might work)
2. A timings waveform
3. The right signals and 8 or 16 bit parallel data. This is already provided by **epdiy**

Having already the pin-out and right FPC connector, checking with your multimeter that the right voltages are applied to the right pins is a good start. But sometimes if you don't have the right timings, you can also get in trouble, although most of the times trying with one of the open source waveforms will work (I mostly try with epdiy_ED097TC2)

But you really do not start with epdiy trying a new still unsupported panel just because you found a nice offer in Aliexpress. That's really not the way. It's like starting to drive with a McLaren just because there was one abandoned and at a good price in your local car dealer.
The best way to start is to get a cheap display, a 6" or 9.7" and try to make the most of your idea, get the feeling of how far you can go with the component. And only after grasping a bit the electronics of it, getting to know KiCad and the [basics of driving Eink displays](http://essentialscrap.com/eink/), then you might take a deep dive in designing an adapter and connecting a new panel. 

That are my thoughts since many times I'm getting this kind of requests, and believe me, I also like to test new panels.
I drove one of the first times [Wuxi Weifeng DES](https://en.wf-tech.com/pro_1341.html) panels, an Eink that I had like 2 years in my office, until v7 and the 16 bit support came to drive it (Now Valentin owns this since I sent this to him to test color implementation)
![DES eink from Wuxi tech](https://pbs.twimg.com/media/GKAzHMgXQAA3mvv?format=jpg&name=large)

But what we cannot do is with every request, take the time to make adapters and buy new panels, just because someone has the wish to drive that particular Eink. This is a collaborative and open source effort. So if you really want hard to try new panels you also have to take **your own risks**.

The idea is first try it, learn the driving part, then implement new epaper panels. Do not start with epdiy with an unknown panel.
And don't get me wrong is probably an awesome experience to start driving with a McLaren or a Porsche, but be aware that you might end up [wrecking the car](https://supercarblondie.com/elon-musk-mclaren-f1) if you press the accelerator too hard!

related to #290 #296 CC please also correct me if I'm wrong @vroland @mcer12 @mickeprag @schuhumi

**Comment by vroland** [comment]
I agree, I'd always recommend starting with a known good display to test that everything is working and build your knowledge. If you do want to try a new panel, it's great if you can acquire a working device from somewhere, maybe a used one with a broken display that you can replace, etc. That way, you can take measurements and see if the display has some peculiarity.
